src/diy/imu-comparison.md

```diff
@@ -46,6 +47,24 @@ Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"
 
 > Please note, if you are looking for BNO085s SlimeVR may have [spares for sale that can be found here](https://shop.slimevr.dev/products/slimevr-imu-module-bno085).
 
+---
+## LSM6DSV
+The LSM6DSV is a new and <b>experimental</b> IMU for DIY SlimeVR.
+Offers very good reset times even in bad magnetic enviroments and for fast movements, dancing. Low availability because the chip is new. 
+
+|Reset time |Cost |Availability|Build quality|
+|:---------:|:---:|:----------:|:-----------:|
+|60 min     |~$15 |Insufficient|Excellent    |
+
+Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-half-o"></i>
+
+|Pros          |Cons                                   |
+|--------------|---------------------------------------|
+|Accurate               |Low availability              |
+|Reliable               |Experimental Beta Firmware    |
+|Consistent Reset times |                              |
+|Single time calibration|                              |
```

Comment:
```suggestion
|Pros                   |Cons                                                          |
|-----------------------|--------------------------------------------------------------|
|Accurate               |[Requires manual calibration the first time](#imu-calibration)|
|Reliable               |Low availability                                              |
|Single time calibration|Experimental Beta Firmware                                    |
|                       |Performance data not taken from survey                        |
```

I wouldn't keep the "consistent reset times" line since performance is so heavily dependent on calibration.